ShellScript/Bash.sublime-syntax

```diff
@@ -1991,6 +1995,10 @@ contexts:
     - match: \\[$`"\\]
       scope: constant.character.escape.shell
 
+  string-commons:
+    - match: ':(?!$|\s|//)'
+      scope: punctuation.separator.sequence.shell
```

Comment:
Having a `string-commons` next to `string-prototype` feels a bit duplicate.

I'd therefore suggest to 

1. either rename it to something which expresses the meaning of the contained pattern more precisly 
2. or to move this pattern to `string-prototype`. It was added as empty map as there was no content yet. It needs to be extended via `- meta_prepend` by extending syntaxes anyway.

2nd solution would also add highlighting of path separators in ansi-c strings but exclude it from heredocs. Not sure about ansi-c strings, but I'd probably would not highlight colons in heredocs as those may contain any kind of larger text blocks with higher risk of false positives, woudln't they?

Reply:
agreed with all your points, so I went for the 1st solution of renaming the new context :)